src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultValue = "";
+
+class SearchForm extends Component {
+  state = {
+    defaultValue: defaultValue,
+    onSearch: (value) => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.defaultValue = props.defaultValue || defaultValue;
+    this.state.onSearch = props.onSearch;
+  }
+
+  search = () => {
+    let searchInput = document.getElementById("searchInput");
```

Comment:
Let's not use getting elements from DOM. It's better to use component state in React.

Please, review "Controlled and uncontrolled components", "DOM vs Virtual DOM".
https://ru.reactjs.org/docs/uncontrolled-components.html
https://goshacmd.com/controlled-vs-uncontrolled-inputs-react/
https://ru.reactjs.org/docs/faq-internals.html#gatsby-focus-wrapper

Reply:
Made SearchComponent a controlled one, the input is cleared after the form submitted (199e1d5e57dd8e523ce5eca19af3f65fc5d22fc3)
Empty value search is not allowed (adada93ae61db74c07c0909cf2cc5150155eacc3)